### PR TITLE
Update help_faq.md on Login

### PR DESCRIPTION
**Pull request by h-m-f-t**
The Login.gov FAQ was missing a key element to explain what we're using Login for and why authentication with it is necessary but insufficient to get a .gov domain.

### CODE REVIEW

**Review by Katherine-Osos — COMMENTED** [review]
The new wording you added "a person is who they say they are" is stated in the second bullet right above that sentence. Did you want it to be repeated?  (just double-checking that you didn't miss that bullet)

**Comment by h-m-f-t** [comment]
😅 You're totally right. Bringing this back for revision.

File: pages/help/help_faq.md [hunk 1]
```diff
@@ -35,12 +35,12 @@ Get more [help with domain management](../domain-management).
...
 
-The state of the art in identity verification is to use attested information (like a license or a SSN). This is why we use Login.gov. However, there is no technology to confirm that a person actually represents a government organization. We manually verify that claim as part of our investigation.
+To prevent someone from using public information to impersonate a government or an official, as well as discourage unauthorized individuals or ineligible organizations from requesting a domain, we verify the identity of new domain requestors using Login.gov. Once your identity has been confirmed, we then conduct a manual review to assess whether you actually work for, or on behalf of, a government.
+
+Login.gov is a secure, government website that adheres to the highest standards in data protection. Most of the data you submit is not stored. You can learn more about the [privacy and security measures](https://login.gov/policy/) taken to keep your information safe.
```
Comment (Katherine-Osos) [inline pages/help/help_faq.md:43]:
```suggestion
Login.gov is a secure, government website that adheres to the highest standards in data protection. Most of the data you submit is not stored. You can learn more about the [privacy and security measures](https://login.gov/policy/){.usa-link--external} taken to keep your information safe.
```

Comment (h-m-f-t) [reply pages/help/help_faq.md]: Excellent, thanks! Added.

**Review by Katherine-Osos — APPROVED** [review]
I added the "external link" icon. Other than that, looks good!

**Pull request merged**